testsuite/integration-arquillian/tests/base/src/test/resources/META-INF/keycloak-server.json

```diff
@@ -209,6 +209,12 @@
         }
     },
 
```

Comment:
Shouldn't the `publicKeyStorage` config be present here too?

Reply:
You are right, added.